### DISCUSSION TITLE
How to list property constraints?

### DISCUSSION THREAD

**Discussion by adubkov** [ArcadeData/arcadedb#882] [Q&A]
I can't find it in the documentation. Is there a way to list which constraints do properties have?
`info types` seems to return only `id`, `name` and `type`:
```
1[{id=0, name=name, type=STRING, custom={}}]
```

Related issue https://github.com/ArcadeData/arcadedb/issues/888

**Comment by lvca** [comment]
Your idea in the issue #888 works for me, I'm going to assign it for the next release in March.
